lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1156,3 +1156,42 @@ ONNXModelLoader::ONNXModelLoader(const std::string &modelDescFilename,
     EXIT_ON_ERR(setup());
   }
 }
+
+ONNXModelLoader::ONNXModelLoader(const std::string &modelDescFilename,
```

Comment:
This is mostly copy pasted from previous constructor, Maybe just factor the `setup` lambda into a separate function with a bool input specifiying how to treat inputs and call it from each constructor or something like that?

Reply:
I can re-use constructor if names and types are empty then use loadInputs().